math/postfix_calculator.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ *
+ * Copyright 2020 @author RameshwarPawar
+ *
+ * @file
+ * program to calculate prosfix expression
+ *
+ */
+#include<iostream>
+#include<stack>
+#include<cstring>
+
+void infix_to_postfix(char *in, char *post);
+bool is_operand(char ch);
+int priority(char opr);
+int postfix_evaluation(char *post);
+int calculate(int op1, int op2, char opr);
+
+void infix_to_postfix(char *in, char *post) {
+int j=0;
+std::stack<int> s;
+
+for(int i = 0 ; in[i] != '\0' ; i++ ) {
```

Comment:
code is not structured with spaces. Follow Google syntax style. If you use VSCode, the repository already has a syntax style format specifier for it